core/trino-main/bin/check_webui.sh

```diff
@@ -13,20 +13,20 @@ WEBUI_ROOT="$(pwd)/${BASH_SOURCE%/*}/../src/main/resources/webapp"
 # Fail if running the command to generate the `dist` folder again produces different results.
 # The lockfile is generated code also, and must go through the same process.
 
-pushd $(mktemp -d)
+pushd "$(mktemp -d)"
 
 cp "${WEBUI_ROOT}/src/yarn.lock" .
 cp -r "${WEBUI_ROOT}/dist" .
 
-yarn --cwd ${WEBUI_ROOT}/src/ install
+yarn --cwd "${WEBUI_ROOT}"/src/ install
```

Comment:
put entire path to `"`. i.e. `"${WEBUI_ROOT}/src/"`, please do the same for others.